service/src/main/java/uk/gov/hmcts/reform/fpl/model/OtherProceeding.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.reform.fpl.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class OtherProceeding {
+    private final String onGoingProceeding;
+    private final String proceedingStatus;
+    private final String caseNumber;
+    private final String started;
+    private final String ended;
+    private final String ordersMade;
+    private final String judge;
+    private final String children;
+    private final String guardian;
+    private final String sameGuardianNeeded;
+    private final String sameGuardianDetails;
+
+    @JsonCreator
```

Comment:
confused as to why this is needed? variable names all seem to match the json values?

Reply:
Yeah really not needed. Removing it.